lib/QubitCsvImport.class.php

```diff
@@ -35,8 +35,20 @@ class QubitCsvImport
   public $indexDuringImport = false;
   public $doCsvTransform = false;
 
-  public function import($csvFile, $type = null)
+  public function import($csvFile, $type = null, $csvOrigFileName = null)
   {
+    if (null === $csvOrigFileName)
```

Comment:
Maybe you can use PHP's ternary operator + shorthand syntax `$csvOrigFileName = basename($csvOrigFileName ?: $csvFile);` to condense this. But maybe we want it as-is so the comments are clearer, I'll leave it up to you.